_posts/en/newsletters/2019-01-29-newsletter.md

```diff
@@ -0,0 +1,219 @@
+  guessing game, Bitcoin's proof of work doesn't sound compelling, but
+  this answer from one of Bitcoin StackExchange's top 30 experts,
+  Chytrik, provides a simple analogy that captures the essence of proof
+  of work and how it helps keep Bitcoin transactions secure.
+
+Optech also congratulates and thanks Pieter Wuille, who this month
+became the [all-time top-voted contributor][top bse] to Bitcoin
+StackExchange.
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+and [libsecp256k1][libsecp256k1 repo].*
+
+- [Bitcoin Core #14955][] switches the Random Number Generator (RNG)
+  used from OpenSSL to Bitcoin Core's own implementation, although RNG
+  output gathered by Bitcoin Core is fed out to OpenSSL and then read
+  back in when the program needs strong randomness.  This moves Bitcoin
+  Core a little closer to no longer needing to depend on OpenSSL, as
+  that dependency has caused security issues in the past.  The PR
+  description and the code changes are very well documented for anyone
+  concerned about the safety of this change.
+
+- [Bitcoin Core #14353][] adds a new REST call
+  `/rest/blockhashbyheight/` for fetching the block in the current best
+  block chain based on its height (how many blocks after the Genesis
+  Block it is).
+
+- [Bitcoin Core #15193][] sets the `whitelistforcerelay` configuration
+  option to off by default.  When enabled, this option cause a node to
+  relay transactions from its manually whitelisted peers and clients
+  even if those transactions violate node policy or consensus rules.
+  This could cause the relaying node, rather than the origin node or
+  client, to be banned by its peers, so it's better to default to
+  turning this option off.  Developers are also asking anyone using this
+  feature to [contact][contact core] them so that they know it's not an
+  unused option that should be deprecated in the future.
+
+- [LND #2314][] adds a chain notifier subserver, allowing services to
+  receive notification about changes to the best block chain---such as
+  when new blocks are received, when transactions get confirmed, and
+  whether or not an input has been spent.
+
+- [LND #2405][] allows different autopilot heuristics to be combined into
+  a single score for each node to which you could connect.  The higher a
+  score, the more it's expected that opening a channel to that node will
+  increase the connectivity of your node (according to various
+  characteristics).
+
+- [LND #2350][] adds a `query` option for the autopilot that accepts a
+  list of LN nodes and returns the scores for those nodes indicating how
+  good a candidate they are for opening a channel to them.
+
+- [LND #2460][] adds support for the `max_htlc` field in channel
+  updates.  This feature allows lite clients and pruned nodes to learn
```

Comment:
I have the slightest of preferences for 'light client' over 'lite client'. I see we've used both when referring to Bitcoin P2P light clients. Do you have a preference for house style?

Reply:
That's my fault for changing my own style a couple months back after I started working on some comparison tables between full nodes and lite clients, where there shorter spelling helped condense the tables.  After using it for a bit, I came to rather enjoy the subtle snub implied by "lite", as if lite clients are to full nodes the same kind of unsatisfying as lite food and beverages are to the full-calorie products they mimic.

That's not very charitable of me, and it might be a kind of manipulative use of language on my part that should be avoided in honest writing, so I'll let you make the final call on what spelling we use.

Reply:
I also thought about "light client" vs "lite client" when reading the newsletter.

FWIW, my preference is "light client".